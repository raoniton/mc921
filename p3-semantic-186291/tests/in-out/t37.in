class Main {
    public static void main(String[] args) {
        int[] y = {1, 2, '3', 4, 5};
    }
}