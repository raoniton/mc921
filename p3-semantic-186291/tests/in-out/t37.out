SemanticError: '3' of type(char) is incompatible with type(int) of array @ 3:26
